e2e/fiat_tf_test.go

```diff
@@ -0,0 +1,209 @@
+	require.ErrorContains(t, err, "you are not the owner: unauthorized")
+
+	showPauserRes, err = showPauser(ctx, val)
+	require.NoError(t, err, "failed to query show-pauser")
+	require.Equal(t, expectedGetPauserResponse.Pauser, showPauserRes.Pauser)
+
+	// ACTION: Update Pauser from blacklisted owner account
+	// EXPECTED: Success; pauser updated
+	// Status:
+	// 	Pauser: newPauser1
+
+	blacklistAccount(t, ctx, val, nw.fiatTfRoles.Blacklister, nw.fiatTfRoles.Owner)
+
+	_, err = val.ExecTx(ctx, nw.fiatTfRoles.Owner.KeyName(), "fiat-tokenfactory", "update-pauser", newPauser2.FormattedAddress())
+	require.NoError(t, err, "failed to broadcast update-pauser message")
+
+	showPauserRes, err = showPauser(ctx, val)
+	require.NoError(t, err, "failed to query show-pauser")
+	expectedGetPauserResponse = fiattokenfactorytypes.QueryGetPauserResponse{
+		Pauser: fiattokenfactorytypes.Pauser{
+			Address: string(newPauser2.FormattedAddress()),
+		},
+	}
+	require.Equal(t, expectedGetPauserResponse.Pauser, showPauserRes.Pauser)
+
+	unblacklistAccount(t, ctx, val, nw.fiatTfRoles.Blacklister, nw.fiatTfRoles.Owner)
+
+	// ACTION: Update Pauser to blacklisted Pauser account
+	// EXPECTED: Success; pauser updated
+	// Status:
+	// 	Pauser: newPauser2
+
+	w = interchaintest.GetAndFundTestUsers(t, ctx, "new-pauser-3", math.OneInt(), noble)
+	newPauser3 := w[0]
+
+	blacklistAccount(t, ctx, val, nw.fiatTfRoles.Blacklister, newPauser3)
+
+	_, err = val.ExecTx(ctx, nw.fiatTfRoles.Owner.KeyName(), "fiat-tokenfactory", "update-pauser", newPauser3.FormattedAddress())
+	require.NoError(t, err, "failed to broadcast update-pauser message")
+
+	showPauserRes, err = showPauser(ctx, val)
+	require.NoError(t, err, "failed to query show-pauser")
+	expectedGetPauserResponse = fiattokenfactorytypes.QueryGetPauserResponse{
+		Pauser: fiattokenfactorytypes.Pauser{
+			Address: string(newPauser3.FormattedAddress()),
+		},
+	}
+	require.Equal(t, expectedGetPauserResponse.Pauser, showPauserRes.Pauser)
+}
+
+func TestFiatTFPause(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	t.Parallel()
+
+	ctx := context.Background()
+
+	nw := nobleSpinUp(t, ctx, true)
+	noble := nw.chain
+	val := noble.Validators[0]
+
+	// ACTION: Pause TF from an account that is not the Pauser
+	// EXPECTED: Request fails; TF not paused
+
+	w := interchaintest.GetAndFundTestUsers(t, ctx, "alice", math.OneInt(), noble)
+	alice := w[0]
+
+	_, err := val.ExecTx(ctx, alice.KeyName(), "fiat-tokenfactory", "pause")
+	require.ErrorContains(t, err, "you are not the pauser: unauthorized")
+
+	showPausedRes, err := showPaused(ctx, val)
+	require.NoError(t, err, "error querying for paused state")
+	expectedPaused := fiattokenfactorytypes.QueryGetPausedResponse{
+		Paused: fiattokenfactorytypes.Paused{
+			Paused: false,
+		},
+	}
+	require.Equal(t, expectedPaused, showPausedRes)
+
+	// ACTION: Pause TF from a blacklisted Pauser account
```

Comment:
Seems that assertions of the expected result are missing from this case and the one below?

Reply:
We do an assertion inside the `pauseFiatTF` function: https://github.com/noble-assets/noble/blob/149a7f6f8748b761bc2ce2108da09c27916b101a/e2e/utils.go#L421-L433